xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -749,13 +751,41 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
 
     CLog::Log(LOGINFO, "CDVDVideoCodecAndroidMediaCodec::Open Testing codec: {}", m_codecname);
 
+    // There is some confusion for video/VC1 vs. video/wvc1 especially on Sony devices, while
+    // IANA defines VC-1 as video/vc1, Android API defines it as video/wvc1 - be nice and test it
+    const bool isVC1 = StringUtils::Contains(m_codecname, "vc1", true);
+
     CJNIMediaCodecInfoCodecCapabilities codec_caps = codec_info.getCapabilitiesForType(m_mime);
     if (xbmc_jnienv()->ExceptionCheck())
     {
       // Unsupported type?
       xbmc_jnienv()->ExceptionDescribe();
       xbmc_jnienv()->ExceptionClear();
-      continue;
+      if (!isVC1)
+        continue;
+      else
```

Comment:
You do not need an else here, because you continued in the line before. Safes another two curly braces. 😬

Reply:
Okay, did that - yeah readability now becomes a bit questionable, but okay - here we go.